source/reference/verification.txt

```diff
@@ -0,0 +1,211 @@
+
+####################
+Verify Data Transfer
+####################
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Following a sync, verify the successful transfer of your data before you switch
+your application from using the source cluster to the destination cluster. You
+can verify your data transfer using document counts, hash comparison, document
+comparison, or the Migration Verifier.
+
+Use Cases
+=========
+
+You should verify your data after every sync. This is important in
+cases where you plan to move your application load from the source to the
+destination cluster.
+
+Verification methods include the comparison of
+:ref:`document counts <c2c-verify-doc-counts>`,
+comparison of :ref:`database hashes <c2c-verify-hash-comp>`,
+scripting :ref:`document comparisons <c2c-verify-doc-comp>`,
+and the :ref:`Migration Verifier <c2c-verify-migration-verifier>`.
+The specific method you use to verify your data depends on your application
+workload and the complexity of the data.
+
+Details
+=======
+
+.. _c2c-verify-doc-counts:
+
+Document Counts
+---------------
+
+The most basic method of verifying sync is by comparing the number of documents
+in each synced collection on the source cluster to the number on the
+destination cluster.
+
+Compare the results on each cluster:
+
+- If the document count is different,  not all changes synced to the
+  destination cluster.
+
+- If the document count is the same, the destination cluster has the same
+  number of documents. This doesn’t guarantee a successful sync of all changes
+  to those documents, except in cases where you have an insert-only workload.
+
+To verify that the source and destination cluster have the same number of
+documents, run the :method:`~db.collection.countDocuments` method for each synced
+collection on each cluster and ensure that they have the same number of
+documents.
+
+.. io-code-block::
+
+   .. input::
+      :language: javascript
+
+      db.us_accounts.countDocuments()
+
+   .. output::
+      :language: javascript
+
+      345
+
+
+.. _c2c-verify-hash-comp:
+
+Hash Comparison
+---------------
+
+You can verify sync by comparing MD5 hashes of collections synced from the
+source cluster to the destination cluster. While hash comparison does ensure
+that the destination cluster has received all changes from the source, the
+:dbcommand:`dbHash` command locks the cluster, preventing additional writes
+until its operations complete.
+
+.. note::
+
+   Hash comparison is not possible with sharded clusters. It also does not
+   work for standalone servers and replica sets that use MongoDB 4.4 or earlier
+   releases, since the field order can vary.
+
+To verify that the source and destination cluster have the same hashes, execute
```

Comment:
We use "run" for the previous section `countDocuments()`.

```suggestion
To verify that the source and destination cluster have the same hashes, run
```

Reply:
Fixed